rust/parallel_calls/.gitignore

```diff
@@ -0,0 +1,25 @@
+# Various IDEs and Editors
```

Comment:
Do you need a specific `.gitignore` for this example? We have one at the top level that should cover things I believe and no other examples have a specific `.gitignore`.

Reply:
Nice catch, thanks!